src/analysis/normed_space/operator_norm.lean

```diff
@@ -1,193 +1,174 @@
 
--- Define the subspace of bounded linear maps.
-section bounded_linear_maps
+noncomputable theory
 set_option class.instance_max_depth 50
 
-
-variables [hnfk : normed_field k] [normed_space k E] [normed_space k F]
-include hnfk
-
-def bounded_linear_maps : subspace k (E → F) :=
-{ carrier := {A : E → F | is_bounded_linear_map k A},
+def bounded_linear_map : subspace k (E → F) :=
+{ carrier := {f : E → F | is_bounded_linear_map k f},
   zero := is_bounded_linear_map.zero,
-  add := assume A B, is_bounded_linear_map.add,
-  smul := assume c A, is_bounded_linear_map.smul c }
+  add := λ _ _, is_bounded_linear_map.add,
+  smul := λ _ _, is_bounded_linear_map.smul _ }
 
--- introduce the local notation L(E,F) for the set of bounded linear maps.
-local notation `L(` E `,` F `)` := @bounded_linear_maps k E F _ _ _
+namespace bounded_linear_map
+
+notation β ` →L[`:25 α `] ` γ := @bounded_linear_map α β γ _ _ _
 
 /-- Coerce bounded linear maps to functions. -/
-instance bounded_linear_maps.to_fun : has_coe_to_fun $ L(E,F) :=
+instance to_fun : has_coe_to_fun $ E →L[k] F :=
 {F :=  λ _, (E → F), coe := (λ f, f.val)}
 
-@[extensionality] theorem ext {A B : L(E,F)} (H : ∀ x, A x = B x) : A = B :=
-set_coe.ext $ funext H
+@[extensionality] theorem ext {f g : E →L[k] F} (h : ∀ x, f x = g x) : f = g :=
+set_coe.ext $ funext h
 
-def to_linear_map (A : L(E, F)) : linear_map _ E F :=
-{to_fun := A.val, ..A.property}
+theorem ext_iff {f g : E →L[k] F} : f = g ↔ ∀ x, f x = g x :=
+⟨λ h x, by rintro; rw h, ext⟩
 
-@[simp] lemma bounded_linear_maps.map_zero {A : L(E, F)} : A (0 : E) = 0 :=
-(to_linear_map A).map_zero
+variables (c : k) (f g: E →L[k] F) (u v: E)
 
-@[simp] lemma bounded_linear_maps.coe_zero {x : E} : (0 : L(E, F)) x = 0 := rfl
+def to_linear_map : linear_map _ E F :=
+{to_fun := f.val, ..f.property}
 
-@[simp] lemma bounded_linear_maps.smul_coe {A : L(E, F)} {x : E} {c : k} :
-  (c • A) x = c • (A x) := rfl
+-- make some straightforward lemmas available to `simp`.
+@[simp] lemma map_zero : f (0 : E) = 0 := (to_linear_map _).map_zero
+@[simp] lemma map_add  : f (u + v) = f u + f v := (to_linear_map _).map_add _ _
+@[simp] lemma map_sub  : f (u - v) = f u - f v := (to_linear_map _).map_sub _ _
+@[simp] lemma map_smul : f (c • u) = c • f u := (to_linear_map _).map_smul _ _
+@[simp] lemma map_neg  : f (-u) = - (f u) := (to_linear_map _).map_neg _
 
-@[simp] lemma bounded_linear_maps.zero_smul {A : L(E, F)} : (0 : k) • A = 0 :=
-by ext; simp
+@[simp] lemma coe_zero : (0 : E →L[k] F) = 0 := rfl
 
-@[simp] lemma bounded_linear_maps.one_smul {A : L(E, F)} : (1 : k) • A = A :=
-by ext; simp
+@[simp] lemma zero_apply : (0 : E →L[k] F) u = 0 := rfl
+@[simp] lemma smul_apply : (c • f) u = c • (f u) := rfl
+@[simp] lemma neg_apply  : (-f) u = - (f u) := rfl
 
-/-- Bounded linear maps are ... bounded -/
-lemma exists_bound (A : L(E,F)) : ∃ c, 0 < c ∧ ∀ x : E, ∥A x∥ ≤ c * ∥x∥ := A.property.bound
+@[simp] lemma zero_smul : (0 : k) • f = 0  := by ext; simp
```

Comment:
The recommended style is now `by { ext, simp }` if there are no multiple goals. Here and several times below.